doc/src/pair_sw.rst

```diff
@@ -24,13 +24,17 @@ Syntax
    pair_style style keyword values
 
 * style = *sw* or *sw/mod*
-* keyword = *maxdelcs*
+* keyword = *maxdelcs* or *threebody*
 
   .. parsed-literal::
 
        *maxdelcs* value = delta1 delta2 (optional)
          delta1 = The minimum thershold for the variation of cosine of three-body angle
          delta2 = The maximum threshold for the variation of cosine of three-body angle
+       *threebody* value = *on* or *off* (optional)
+         on (default) = Compute both the three-body and two-body terms of the potential
+         off = Compute both only the two-body terms of the potential
```

Comment:
"Compute both only" -> "Compute only" and "terms" -> "term"

Reply:
Resolved by 9a05cd3